aten/src/ATen/native/xnnpack/RegisterOpContextClass.cpp

```diff
@@ -60,6 +60,15 @@ torch::jit::class_<Conv2dOpContext> register_packed_conv2d_op_context_class() {
 static auto linear_op_context_class = register_packed_linear_op_context_class();
 static auto conv2d_op_context_class = register_packed_conv2d_op_context_class();
 
+// TODO: These should probably be in an xnnpack namespace?
```

Comment:
The choice of not using xnnpack namespace was explicit as change was made as part of this PR: https://github.com/pytorch/pytorch/pull/35010. Relevant issue: https://github.com/pytorch/pytorch/issues/34913

Reply:
thanks, I'll fix this